src/wp-includes/fonts.php

```diff
@@ -140,7 +150,7 @@ function wp_get_font_dir( $defaults = array() ) {
 	 *
 	 * @param array $defaults The original fonts directory data.
 	 */
-	return apply_filters( 'font_dir', $defaults );
+	return doing_filter( 'font_dir' ) ? $defaults : apply_filters( 'font_dir', $defaults );
```

Comment:
Hmm 🤔, only one of the `doing_filter( 'font_dir' )` should be needed. Let's evaluate.

If `doing_filter( 'font_dir' )`, then `wp_get_font_dir()` will have already made its changes.

- Bailing early should be safe.
- Bailing late would overwrite the previously filtered `$defaults`.

@hellofromtonya @matiasbenedetto Can I get a confidence check on that?

Reply:
Nevermind, [the alternative approach](https://github.com/WordPress/gutenberg/pull/58839).